blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -86,6 +86,9 @@ Changes
 Fixes
 =====
 
+- Store current ``created`` versions at new table partitions instead of using
```

Comment:
would it be better to formulate this based on the behaviour we fixed for the user?

Reply:
what do you have in mind?